src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -12500,6 +12886,48 @@ public void SynthesizedDelegateTypes_ParamsArray_DifferentElementTypes()
                 """).VerifyDiagnostics();
         }
 
+        [Fact]
+        public void SynthesizedDelegateTypes_ParamsArray_LeastOverriddenMethod()
```

Comment:
>SynthesizedDelegateTypes_ParamsArray_LeastOverriddenMethod [](http://example.com/codeflow?start=20&length=58)

This test and the one below doesn't test the interesting scenario, i.e. a mismatch in params between abstract and override methods. You can verify that by comparing ```IsParams``` values on the parameter symbols. The comment asking to test the scenarios was also including information about how to create a mismatch situation. In addition the test should verify presence or lack of params in the inferred delegate